CMSgov/design-system
# [WNMGDS-1133] e2e test cleanup

## Summary
Removing browser tests that just rendered a component to the screen. Renamed files to be `.a11y.test`

### Added
* Some components didn't have a11y tests, so I added files for `Accordion`
_Note: some components (`StepList` & `FilterChip`) still don't have a11y tests because violations were flagged when the test files were added. I split the work to add a11y tests and resolve the violations into separate tickets_

### Changed
* Updated build scripts and documentation to reflect updated `test:a11y` command
* Updated file names from `.e2e.test` --> `.a11y.test`
* Updated `chromedriver` dependency so selenium would run

### Removed
* browser tests that just rendered a component to the page. This can be done through visual regression

## How to test
1. Pull down branch
2. Run `yarn install` to get updated chromedriver
3. Run `yarn test` and/or `yarn test:a11y` to confirm that accessibility tests run & pass
